dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/NacosRegistry.java

```diff
@@ -406,9 +418,11 @@ private void subscribeEventListener(String serviceName, final URL url, final Not
      */
     private void notifySubscriber(URL url, NotifyListener listener, Collection<Instance> instances) {
         List<Instance> healthyInstances = new LinkedList<>(instances);
-        // Healthy Instances
-        filterHealthyInstances(healthyInstances);
-        List<URL> urls = buildURLs(url, healthyInstances);
+        if (healthyInstances.size() > 0) {
```

Comment:
If healthyInstances is empty, this still works? Not sure, could u pls check it.
I believe we should make our code as clean as possible :)

Reply:
Same as above.